corporate/lib/stripe.py

```diff
@@ -3442,14 +3443,19 @@ def update_free_trial_invoice_with_licenses(
         # the updated quantity.
         stripe_invoice = stripe.Invoice.retrieve(last_sent_invoice.stripe_invoice_id)
         assert stripe_invoice.status == "open"
+        assert isinstance(stripe_invoice.customer, str)
+        assert stripe_invoice.statement_descriptor is not None
+        assert stripe_invoice.metadata is not None
         invoice_items = stripe_invoice.lines.data
         # Stripe does something weird and puts the discount item first, so we need to reverse the order here.
         invoice_items.reverse()
         for invoice_item in invoice_items:
+            assert invoice_item.description is not None
             price_args: PriceArgs = {}
             # If amount is positive, this must be non-discount item we need to update.
             if invoice_item.amount > 0:
                 assert invoice_item.price is not None
+                assert invoice_item.price.unit_amount is not None
```

Comment:
These four `assert`s want the most review attention, I think.

Reply:
@amanagr can you do post-commit review on these? They seem plausible to me.